libs/exo-deno/src/test_js/through_rust_fn.js

```diff
@@ -6,11 +6,12 @@
 // As of the Change Date specified in that file, in accordance with
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
+import { rust_impl, async_rust_impl } from "test:through_rust";
 
 export function syncUsingRegisteredFunction(value) {
-  return Deno[Deno.internal].core.ops.rust_impl(value)
+  return rust_impl(value)
 }
 
-export async function asyncUsingRegisteredFunction(value) {
-  return Deno[Deno.internal].core.opAsync("async_rust_impl", value)
+export function asyncUsingRegisteredFunction(value) {
```

Comment:
This needs to be `async`. I think I dropped the `async` in a previous commit when experimenting.